src/lib/filterables.ts

```diff
@@ -2,6 +2,97 @@ import resolveItems from './util/resolveItems';
 import { warmGear } from '../commands/Minion/wt';
 import { gracefulItems } from './skilling/skills/agility';
 import { Eatables } from './eatables';
+import Crafting from './skilling/skills/crafting/crafting';
+import Fletching from './skilling/skills/fletching/fletching';
+
+const ores = resolveItems([
+	`Copper ore`,
+	`Tin ore`,
+	`Iron ore`,
+	`Blurite ore`,
+	`Silver ore`,
+	`Coal`,
+	`Elemental ore`,
+	`Gold ore`,
+	`Mithril ore`,
+	`Lovakite ore`,
+	`Adamantite ore`,
+	`Runite ore`
+]);
+
+const bars = resolveItems([
+	`Bronze bar`,
+	`Blurite bar`,
+	`Iron bar`,
+	`Elemental metal`,
+	`Steel bar`,
+	`Primed bar`,
+	`Primed mind bar`,
+	`Lovakite bar`,
+	`Mithril bar`,
+	`Adamantite bar`,
+	`Runite bar`
+]);
+
+const smithingMisc = resolveItems([
+	`Shield left half`,
+	`Shield right half`,
+	`Dragon metal shard`,
+	`Dragon metal slice`,
+	`Godsword blade`,
+	`Godsword shards 1 & 2`,
+	`Godsword shards 1 & 3`,
+	`Godsword shards 2 & 3`,
+	`Godsword shard 1`,
+	`Godsword shard 2`,
+	`Godsword shard 3`,
+	`Armadyl hilt`,
+	`Bandos hilt`,
+	`Saradomin hilt`,
+	`Zamorak hilt`,
+	`Arcane sigil`,
+	`Spectral sigil`,
+	`Elysian sigil`,
+	`Blessed Spirit Shield`,
+	`Smouldering stone`,
+	`Dragon metal lump`,
+	`Draconic visage`,
+	`Skeletal visage`,
+	`Wyvern visage`
+]);
+
+const gems = resolveItems([
+	`Uncut opal`,
+	`Uncut jade`,
+	`Uncut red topaz`,
+	`Uncut sapphire`,
+	`Uncut emerald`,
+	`Uncut ruby`,
+	`Uncut diamond`,
+	`Uncut dragonstone`,
+	`Amethyst`,
+	`Uncut onyx`,
+	`Sapphire`,
+	`Opal`,
+	`Jade`,
+	`Red topaz`,
+	`Emerald`,
+	`Ruby`,
+	`Diamond`,
+	`Dragonstone`,
+	`Onyx`,
+	`Zenyte`,
+	`Uncut zenyte`,
+	`Zenyte shard`
+]);
+
+const craftingItems = Crafting.Craftables.flatMap(item =>
+	Object.keys(item.inputItems).map(key => parseInt(key))
```

Comment:
concat the output item from the craft here so that its also tracked in the final array
```suggestion
	Object.keys(item.inputItems).map(key => parseInt(key)).concat(item.id)
```

unless you specifically think that finished items shouldnt be in this filter which i suppose can be valid idk

Reply:
after second thought having output items in this filter doesnt have much use